spacy/cli/project/dvc.py

```diff
@@ -123,6 +125,11 @@ def update_dvc_config(
             dvc_cmd.append("--always-changed")
         full_cmd = [*dvc_cmd, *deps_cmd, *outputs_cmd, *outputs_nc_cmd, *project_cmd]
         dvc_commands.append(join_command(full_cmd))
+    if not processed_step:
```

Comment:
I realised this error handling was missing when testing this PR. Strictly speaking it could have gone in a separate PR, but because it was so small and I thought it might conflict with this PR I decided to add it in here.